packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -216,6 +216,11 @@ class InspectorController extends DisposableController
     (widgetProperties: [], renderProperties: []),
   );
 
+  /// Whether the implementation widgets are hidden in the widget tree.
+  ValueListenable<bool> get implementationWidgetsHidden =>
```

Comment:
Nit: maybe `hideImplementationWidgets`?

Reply:
I think prefer "hidden" - "hide" makes this sound like it's a function and not a boolean value.